src/components/Display.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Display = ({ result }) => (
+  <div className="result">
+    <p>{result}</p>
+  </div>
+);
+
+Display.defaultProps = {
+  result: 0,
```

Comment:
- [ ] The default value provided to the result props should be a string instead of a number. Use `'0'` instead. Please kindly make this change before merging the branch

Reply:
Thanks, already fixed in the next milestone work